libs/components/tabs/src/lib/modules/sectioned-form/sectioned-form.component.scss

```diff
@@ -47,10 +47,60 @@
   }
 }
 
+:host-context(sky-modal) {
+  .sky-sectioned-form {
+    min-height: 460px;
+  }
+}
+
+:host-context(sky-modal-content) {
+  :host .sky-sectioned-form {
+    margin: var(--sky-compat-sectioned-form-modal-margin);
+  }
+
+  form :host .sky-sectioned-form {
+    margin: var(--sky-compat-sectioned-form-modal-margin);
+  }
+}
+
+:host-context(sky-modal-content) > {
+  :host .sky-sectioned-form {
+    margin: -$sky-margin-plus-half;
+  }
+
+  form :host .sky-sectioned-form {
+    margin: -$sky-margin-plus-half;
+  }
+}
+
 .sky-sectioned-form-tabs-hidden {
   display: none;
 }
 
 .sky-sectioned-form-content-hidden {
   display: none;
 }
+
+@include mixins.sky-theme-modern {
+  :host-context(sky-modal-content) {
+    :host .sky-sectioned-form {
+      margin: var(--sky-compat-sectioned-form-theme-modern-modal-margin);
+    }
+
+    form :host .sky-sectioned-form {
+      margin: var(--sky-compat-sectioned-form-theme-modern-modal-margin);
+    }
+  }
+
+  :host-context(sky-modal-content) > {
+    :host .sky-sectioned-form {
```

Comment:
Since the margin rules are identical, you can consolidate the selectors it so it's not repeated. Same for similar instances above (I'm reviewing this starting with the last file and moving up, sorry):

```
:host-context(sky-modal-content) > {
  :host .sky-sectioned-form,
  form :host .sky-sectioned-form {
    margin: (-$sky-theme-modern-space-sm) (-$sky-theme-modern-space-xl)
      (-$sky-theme-modern-space-xl) (-$sky-theme-modern-space-xl);
  }
}
```